packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -96,9 +97,9 @@ const Template: ComponentStory<typeof DataList> = args => {
 
   return (
     <DataList
-      loading={loadingInitialContent}
-      totalCount={null}
       {...args}
+      loading={args.loading || loadingInitialContent}
```

Comment:
Super minor but can you add this to the breakpoints story and add a title to the default args of the breakpoint story?